app/src/main/java/tttTDD/Java/Human.java

```diff
@@ -1,18 +1,51 @@
 package tttTDD.Java;
 import tttTDD.Java.Interfaces.Player;
 
+import java.util.Scanner;
+
 public class Human implements Player {
-    private String marker;
+    private final String marker;
+    private Scanner playerInput;
+    private int previousMove;
 
     Human(String marker) {
         this.marker = marker;
     }
 
+    Human(String marker, Scanner sc) {
+        this.marker = marker;
+        this.playerInput = sc;
+    }
+
     public String getMarker() {
         return this.marker;
     }
 
+    public int getPreviousMove() {
+        return this.previousMove;
+    }
+
     public boolean isComputer() {
         return false;
     }
+
+    @Override
+    public int move(Board board) {
+        try {
+            int move = playerInput.nextInt();
+            previousMove = move;
+
+            return convertInputToLocation(move);
+        }
+
+        catch (Exception e) {
```

Comment:
Never catch the base exception in checked-in code; it will make debugging unexpected errors extremely frustrating, it obscures what's happening to source code readers, and in many cases (like this one) it's no easier than being specific.